source/jormungandr/jormungandr/interfaces/v1/journey_common.py

```diff
@@ -484,6 +484,16 @@ def __init__(self, output_type_serializer):
             hidden=True,
             help="use/disuse the entrance/exit in journeys computation",
         )
+        parser_get.add_argument(
+            "additional_time_after_first_section_taxi",
```

Comment:
I prefer longer name than this one : additional_time_after_first_section_taxi_as_i_am_tired_of _walking
I am joking. Can it be shorter ?

Reply:
it has been decided a long time ago.... too late to shorten it... :(